core/pkg/server/sender.go

```diff
@@ -641,7 +641,6 @@ func (s *Sender) sendLinkArtifact(record *spb.Record, msg *spb.LinkArtifactReque
 	err := linker.Link()
 	if err != nil {
 		response.ErrorMessage = err.Error()
-		s.logger.CaptureError(err)
```

Comment:
do you want to still see the errors in the debug file?

Reply:
@kptkin Is this right?

Reply:
perfect! thanks!